### ISSUE TITLE
Null reference exception just before starting

### ISSUE THREAD

**Issue by oleg-pshenin** [github-beta/unity-preview#28]
Please fill out the data below and include a zip with the Unity log file and the extension log file.
On Windows, the extension log file is at `%LOCALAPPDATA%\GitHubUnityDebug\github-unity.log`
On macOS, the extension log file is at `~/.local/share/GitHubUnityDebug/github-unity.log`

If you prefer not to submit your log files through the issue tracker, send them
to unity@github.com with the issue number.

Version of Unity (eg. 5.5.0f1):
Version of GitHub for Unity (see console output):
Platform (eg. Windows, macOS, Linux):

What happened

**Comment by oleg-pshenin** [comment]
Unity 5.6.0f3x64
GitHub 0.9.1.0
Win10 x64

> 19:45:17 PM WARN [ 2] <GitInstaller>                      Archive "PlatformResources\windows\git-lfs.zip" missing
> 19:47:39 PM INFO [ 1] <LoginManager>                      Login Username:olegpshenin
> 19:53:00 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:53:00 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 17:21:44 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 17:21:47 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:24:32 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:38:38 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 22:22:10 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 22:28:19 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 22:39:36 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 22:39:36 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 14:55:38 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 14:57:44 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 14:59:54 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:03:32 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:05:18 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:22:53 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:23:30 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:38:20 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:40:20 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 15:56:10 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:11:54 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:17:56 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:18:17 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:18:33 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:19:05 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:19:05 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:19:08 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:19:22 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:19:22 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:19:24 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:23:10 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:24:01 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:24:06 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:24:06 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:24:08 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:25:00 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:25:00 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 18:25:02 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:31:05 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:31:42 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 18:34:36 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:04:57 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:04:57 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:04:59 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:06:37 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:06:42 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:06:42 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:06:45 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:08:25 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:11:42 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:11:44 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:11:44 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:11:46 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:15:42 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:16:29 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:16:33 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:16:33 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:16:35 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0
> 19:20:06 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:20:06 PM WARN [ 1] <RepositoryWatcher>                 Watcher already stopped
> 19:20:08 PM INFO [ 1]                                     Initializing GitHub for Unity version 0.9.1.0

At the moment after clicking the start button and before project starts (during loading), i get next exception:

> NullReferenceException: Object reference not set to an instance of an object
> GitHub.Unity.RepositoryWatcher.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:329)
> GitHub.Unity.RepositoryWatcher.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:337)
> GitHub.Unity.RepositoryManager.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:606)
> GitHub.Unity.RepositoryManager.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:612)
> GitHub.Unity.ApplicationManagerBase.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:153)
> GitHub.Unity.ApplicationManager.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:158)
> GitHub.Unity.ApplicationManagerBase.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:159)
> GitHub.Unity.ApplicationManager.<ListenToUnityExit>m__2 () (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:106)
> UnityEditor.EditorApplication.Internal_PlaymodeStateChanged () (at C:/buildslave/unity/build/artifacts/generated/common/editor/EditorApplicationBindings.gen.cs:332)
> UnityEditor.Toolbar:OnGUI()

It doesn't affect the work, committing and pushing work fine.
Also, have errors with swf.dl (already described in other thread)l, but they appeared much earlier than this issue and I am not sure that they can be connected.

> Failed to load 'Assets/Editor/GitHub/sfw.dll' with error 'The specified module could not be found.
>
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:70)
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:69)
> GitHub.Unity.RepositoryManager:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:178)
> GitHub.Unity.<RestartRepository>c__async0:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:97)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start(<RestartRepository>c__async0&)
> GitHub.Unity.ApplicationManagerBase:RestartRepository()
> GitHub.Unity.ApplicationManager:RestartRepository() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:91)
> GitHub.Unity.<RunInternal>c__async1:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:121)
> System.Threading.Tasks.ThreadPoolTaskScheduler:TaskExecuteWaitCallback(Object)

and

> Failed to load 'Assets/Editor/GitHub/sfw.dll' with error 'The specified module could not be found.
> 
> sfw.net.NativeInterface:.ctor(String)
> sfw.net.NativeInterface:.ctor(String)
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:69)
> GitHub.Unity.RepositoryManager:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:178)
> GitHub.Unity.<RestartRepository>c__async0:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:97)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start(<RestartRepository>c__async0&)
> GitHub.Unity.ApplicationManagerBase:RestartRepository()
> GitHub.Unity.ApplicationManager:RestartRepository() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:91)
> GitHub.Unity.<RunInternal>c__async1:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:121)
> System.Threading.Tasks.ThreadPoolTaskScheduler:TaskExecuteWaitCallback(Object)

**Comment by oleg-pshenin** [comment]
Unity Log:

> NullReferenceException: Object reference not set to an instance of an object
>   at GitHub.Unity.RepositoryWatcher.Dispose (Boolean disposing) [0x00027] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:329 
>   at GitHub.Unity.RepositoryWatcher.Dispose () [0x00003] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:337 
>   at GitHub.Unity.RepositoryManager.Dispose (Boolean disposing) [0x0002b] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:606 
>   at GitHub.Unity.RepositoryManager.Dispose () [0x00003] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:612 
>   at GitHub.Unity.ApplicationManagerBase.Dispose (Boolean disposing) [0x00030] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:153 
>   at GitHub.Unity.ApplicationManager.Dispose (Boolean disposing) [0x00003] in /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:158 
>   at GitHub.Unity.ApplicationManagerBase.Dispose () [0x00003] in /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:159 
>   at GitHub.Unity.ApplicationManager.<ListenToUnityExit>m__2 () [0x00017] in /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:106 
>   at (wrapper delegate-invoke) UnityEditor.EditorApplication/CallbackFunction:invoke_void__this__ ()
>   at UnityEditor.EditorApplication.Internal_PlaymodeStateChanged () [0x00010] in C:\buildslave\unity\build\artifacts\generated\common\editor\EditorApplicationBindings.gen.cs:332 
> UnityEditor.EditorApplication:set_isPlaying(Boolean)
> UnityEditor.Toolbar:TogglePlaying() (at C:\buildslave\unity\build\Editor\Mono\GUI\Toolbar.cs:688)
> UnityEditor.Toolbar:DoPlayButtons(Boolean) (at C:\buildslave\unity\build\Editor\Mono\GUI\Toolbar.cs:497)
> UnityEditor.Toolbar:OnGUI() (at C:\buildslave\unity\build\Editor\Mono\GUI\Toolbar.cs:345)
>  
> (Filename: /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs Line: 329)

> 19:28:01 PM [ 1]  Initializing GitHub for Unity version 0.9.1.0
> UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
> UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
> UnityEngine.Logger:Log(LogType, Object)
> UnityEngine.Debug:Log(Object)
> GitHub.Unity.UnityLogAdapter:OnInfo(String) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/Logging/UnityLogAdapter.cs:21)
> GitHub.Unity.LogAdapterBase:Info(String) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Logging/LogAdapterBase.cs:26)
> GitHub.Unity.Logging:Info(String) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Logging/Logging.cs:68)
> GitHub.Unity.EntryPoint:.cctor() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs:33)
> System.Runtime.CompilerServices.RuntimeHelpers:RunClassConstructor(IntPtr)
> System.Runtime.CompilerServices.RuntimeHelpers:RunClassConstructor(RuntimeTypeHandle) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Runtime.CompilerServices/RuntimeHelpers.cs:101)
> UnityEditor.EditorAssemblies:ProcessEditorInitializeOnLoad(Type) (at C:\buildslave\unity\build\Editor\Mono\EditorAssemblies.cs:136)
> UnityEditor.EditorAssemblies:ProcessInitializeOnLoadAttributes() (at C:\buildslave\unity\build\Editor\Mono\EditorAssemblies.cs:162)
>  
> (Filename: /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/Logging/UnityLogAdapter.cs Line: 21)

> 19:28:02 PM [ 1]  Initializing GitHub for Unity log file: C:\Users\olegp\AppData\Local\GitHubUnityDebug\github-unity.log
> UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
> UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
> UnityEngine.Logger:Log(LogType, Object)
> UnityEngine.Debug:Log(Object)
> GitHub.Unity.UnityLogAdapter:OnInfo(String) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/Logging/UnityLogAdapter.cs:21)
> GitHub.Unity.LogAdapterBase:Info(String) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Logging/LogAdapterBase.cs:26)
> GitHub.Unity.Logging:Info(String) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Logging/Logging.cs:68)
> GitHub.Unity.EntryPoint:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs:46)
> UnityEditor.EditorApplication:Internal_CallUpdateFunctions() (at C:\buildslave\unity\build\artifacts\generated\common\editor\EditorApplicationBindings.gen.cs:249)
>  
> (Filename: /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/Logging/UnityLogAdapter.cs Line: 21)
> 
> Failed to load 'Assets/Editor/GitHub/sfw.dll' with error 'The specified module could not be found.
> '.
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:70)
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:69)
> GitHub.Unity.RepositoryManager:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:178)
> GitHub.Unity.<RestartRepository>c__async0:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:97)
> System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start(<RestartRepository>c__async0&)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start(<RestartRepository>c__async0&)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start(<RestartRepository>c__async0&)
> GitHub.Unity.ApplicationManagerBase:RestartRepository()
> GitHub.Unity.ApplicationManager:RestartRepository() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:91)
> GitHub.Unity.<RunInternal>c__async1:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:121)
> System.Runtime.CompilerServices.MoveNextRunner:InvokeMoveNext(Object)
> System.Security.SecurityContext:Run(SecurityContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Security/SecurityContext.cs:150)
> System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Threading/ExecutionContext.cs:150)
> System.Runtime.CompilerServices.MoveNextRunner:Run()
> System.Threading.Tasks.Task:InnerInvoke()
> System.Threading.Tasks.Task:Execute()
> System.Threading.Tasks.Task:ExecutionContextCallback(Object)
> System.Security.SecurityContext:Run(SecurityContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Security/SecurityContext.cs:150)
> System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Threading/ExecutionContext.cs:150)
> System.Threading.Tasks.Task:ExecuteWithThreadLocal(Task&)
> System.Threading.Tasks.Task:ExecuteEntry(Boolean)
> System.Threading.Tasks.ThreadPoolTaskScheduler:TaskExecuteWaitCallback(Object)
> 
> [C:\buildslave\unity\build\Runtime/Misc/Plugins.cpp line 186] 
> (Filename: /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs Line: 70)
> 
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/.\sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/libsfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/.\libsfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/libsfw
> Failed to load 'Assets/Editor/GitHub/sfw.dll' with error 'The specified module could not be found.
> '.
> sfw.net.NativeInterface:.ctor(String)
> sfw.net.NativeInterface:.ctor(String)
> GitHub.Unity.RepositoryWatcher:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:69)
> GitHub.Unity.RepositoryManager:Initialize() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:178)
> GitHub.Unity.<RestartRepository>c__async0:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:97)
> System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start(<RestartRepository>c__async0&)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start(<RestartRepository>c__async0&)
> System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start(<RestartRepository>c__async0&)
> GitHub.Unity.ApplicationManagerBase:RestartRepository()
> GitHub.Unity.ApplicationManager:RestartRepository() (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:91)
> GitHub.Unity.<RunInternal>c__async1:MoveNext() (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:121)
> System.Runtime.CompilerServices.MoveNextRunner:InvokeMoveNext(Object)
> System.Security.SecurityContext:Run(SecurityContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Security/SecurityContext.cs:150)
> System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Threading/ExecutionContext.cs:150)
> System.Runtime.CompilerServices.MoveNextRunner:Run()
> System.Threading.Tasks.Task:InnerInvoke()
> System.Threading.Tasks.Task:Execute()
> System.Threading.Tasks.Task:ExecutionContextCallback(Object)
> System.Security.SecurityContext:Run(SecurityContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Security/SecurityContext.cs:150)
> System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Threading/ExecutionContext.cs:150)
> System.Threading.Tasks.Task:ExecuteWithThreadLocal(Task&)
> System.Threading.Tasks.Task:ExecuteEntry(Boolean)
> System.Threading.Tasks.ThreadPoolTaskScheduler:TaskExecuteWaitCallback(Object)
> 
> [C:\buildslave\unity\build\Runtime/Misc/Plugins.cpp line 186] 
> (Filename: /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs Line: 69)
> 
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/.\sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/sfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/libsfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/.\libsfw
> Fallback handler could not load library D:/Unity/Editor/Data/Mono/lib/libsfw
> Refresh: detecting if any assets need to be imported or removed ... Refresh: elapses 0.020045 seconds (Nothing changed)
> Refresh: detecting if any assets need to be imported or removed ... Refresh: elapses 0.025661 seconds (Nothing changed)
> Refresh: detecting if any assets need to be imported or removed ... Refresh: elapses 0.035819 seconds (Nothing changed)
> Refresh: detecting if any assets need to be imported or removed ... Refresh: elapses 0.025822 seconds (Nothing changed)

**Comment by StanleyGoldman** [comment]
Hey @olegpshenin thanks for the bug reports.
I'm gonna split these up so we can track them better.

#22
```
NullReferenceException: Object reference not set to an instance of an object
GitHub.Unity.RepositoryWatcher.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:329)
GitHub.Unity.RepositoryWatcher.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/Events/RepositoryWatcher.cs:337)
GitHub.Unity.RepositoryManager.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:606)
GitHub.Unity.RepositoryManager.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/RepositoryManager.cs:612)
GitHub.Unity.ApplicationManagerBase.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:153)
GitHub.Unity.ApplicationManager.Dispose (Boolean disposing) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:158)
GitHub.Unity.ApplicationManagerBase.Dispose () (at /Users/shana/code/work/github/UnityInternal/src/GitHub.Api/ApplicationManagerBase.cs:159)
GitHub.Unity.ApplicationManager.m__2 () (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationManager.cs:106)
UnityEditor.EditorApplication.Internal_PlaymodeStateChanged () (at C:/buildslave/unity/build/artifacts/generated/common/editor/EditorApplicationBindings.gen.cs:332)
UnityEditor.Toolbar:OnGUI()
```

#8 
```
    Failed to load 'Assets/Editor/GitHub/sfw.dll' with error 'The specified module could not be found.
```
